docs/language/accounts.mdx

```diff
@@ -170,6 +170,15 @@ struct PublicKey {
 
     /// Verifies the validity of this public key
     pub fun validate(): Bool
+
+    /// Verifies a signature. Checks whether the signature was produced by signing
+    /// the given tag and data, using this public key and the given hash algorithm
+    pub fun isValid(
```

Comment:
If we're changing names, shouldn't we just use the verb `Verify`, which is the commonly used verb for verifying cryptographic signatures ? We can keep `isValid` for the public key field since the validation there has a different meaning and is not related to signatures.

Reply:
Agreed, `verify` is an even better name 👍

Reply:
`verify` sounds a better name. Changed it here: bf802b8